README.md

```diff
@@ -54,20 +53,18 @@ import wso2/amazonec2;
 amazonec2:AmazonEC2Configuration amazonec2Config = {
     accessKeyId: "",
     secretAccessKey: "",
-    region: "",
-    clientConfig:{}
+    region: ""
 };
 
 amazonec2:Client amazonEC2Client = new(amazonec2Config);
 
 public function main() {
 
-   var describeInstancesResponse = amazonEC2Client->describeInstances();
-   if (reservations is error) {
-       io:println(msg = < string > reservations.detail().message);
+   var describeInstances = amazonEC2Client->describeInstances("i-0b8e13763e642063a");
```

Comment:
Shall we keep this as `describeInstancesResponse` or change to some other non verb identifier?